conu/backend/docker/backend.py

```diff
@@ -48,7 +47,10 @@ def cleanup_containers():
             client.stop(id)
             client.remove_container(id)
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
-        super(DockerBackend, self).__exit__(exc_type, exc_val, exc_tb)
-        if self.cleanup:
-            DockerBackend.cleanup_containers()
+    def cleanup_volumes(self):
+        # TODO implement cleaning of docker volumes
+        pass
+
+    def cleanup_images(self):
```

Comment:
Does this overwrite exception from parent class? Whatwe want to happen when someone sets cleanup policy that is not implemented yet?